internal/oci/runtime_oci.go

```diff
@@ -896,7 +896,7 @@ func (r *runtimeOCI) UpdateContainerStatus(ctx context.Context, c *Container) er
 		metrics.Instance().MetricContainersOOMTotalInc()
 
 		// Collect metric by container name
-		metrics.Instance().MetricContainersOOMInc(c.Name())
+		metrics.Instance().MetricContainersOOMCountTotalInc(c.Name())
```

Comment:
we need to increment the old one too until we finish deprecating it

Reply:
So I added both here:
https://github.com/cri-o/cri-o/blob/5b7e1417fbb95bb8bda996bf96719700fbe4d06b/server/metrics/metrics.go#L420-L434

Assuming that we don't have to touch `internal/oci/runtime_oci.go` file when we want to remove it in course of deprecation.

Reply:
good idea